Bakis/ClientApp/src/app/components/base-event/base-event.component.ts

```diff
@@ -28,8 +28,8 @@ export class BaseEventComponent implements OnInit, ControlValueAccessor {
     private formBuilder: FormBuilder,
     @Inject(MAT_DIALOG_DATA) public data: any,
   ) {
-    const currentYear = new Date().getFullYear();
-    this.minDate = new Date(currentYear);
+    const currentDate = new Date();
+    this.minDate = currentDate;
```

Comment:
I think you could just do this.mindate = new date(datetime.today()) or something like that